Doc/library/sys.rst

```diff
@@ -1060,6 +1062,31 @@ always available.
         This is still called as a fallback if a :data:`meta_path` entry doesn't
         have a :meth:`~importlib.abc.MetaPathFinder.find_spec` method.
 
+.. data:: module_names
+
+   A tuple of strings giving the names of standard library modules.
+
+   The list is the same on all platforms. Modules which are not available on
+   some platforms and modules disabled at Python build are also listed.
+   All module kinds are listed: pure Python, built-in, frozen and extension
+   modules.
+
+   For packages, only sub-packages are listed, not sub-modules. For example,
+   ``concurrent`` package and ``concurrent.futures`` sub-package are listed,
+   but not ``concurrent.futures.base`` sub-module.
+
+   Some special stdlib modules are excluded, like test and private modules.
```

Comment:
Excluding tests is fine but why excluding private modules? They will appear in `sys.modules` and they belong to the standard library.

Reply:
Well, my rationale is to keep the list looking nice for people who displays the list. There is no technical reason for hiding modules like _collections_abc or _pydecimal. I can easily shrink the IGNORE list in Tools/scripts/generate_module_names.py.

Reply:
> Well, my rationale is to keep the list looking nice for people who displays the list.

But then is nor super usefull for people that want to use the list for reporting for example. Imagine you want to report 3rd party modules. You will iterate over `sys.modules` and check if the names are in this list. If we don't include private modules then the reporting will be wrong.